pkg/common/parameters.go

```diff
@@ -143,6 +147,12 @@ func ExtractAndDefaultParameters(parameters map[string]string, driverName string
 			for labelKey, labelValue := range paramLabels {
 				p.Labels[labelKey] = labelValue
 			}
+		case ParameterKeyProvisionedIOPSOnCreate:
+			paramProvisionedIOPSOnCreate, err := ConvertStringToInt64(v)
```

Comment:
Don't we want to use resource quantity here so that we can convert strings like 100k? See k8s.io/apimachinery/pkg/api/resource

Reply:
What are we supporting here? Are we supporting both "100000GB" and "10000MB" or "100000GiB" and "10000MiB" and + "100k GB" ?

Reply:
I think the use case is to support 100k. I'm not sure if there's a k8s resource parsing api to specify that "k" is okay but "GiB" is not. So I think we should just use the permissive api.

eg you could dig and see what cpu does to parse---can you write 2GiB for a CPU request?